examples/gno.land/p/demo/grc/grc20/types.gno

```diff
@@ -55,9 +50,42 @@ type Token interface {
 	TransferFrom(from, to std.Address, amount uint64) error
 }
 
+var (
+	ErrInsufficientBalance   = errors.New("insufficient balance")
+	ErrInsufficientAllowance = errors.New("insufficient allowance")
+	ErrInvalidAddress        = errors.New("invalid address")
+	ErrCannotTransferToSelf  = errors.New("cannot send transfer to self")
+	ErrReadonly              = errors.New("banker is readonly")
+	ErrRestrictedTokenOwner  = errors.New("restricted to bank owner")
+)
+
 const (
 	MintEvent     = "Mint"
 	BurnEvent     = "Burn"
 	TransferEvent = "Transfer"
 	ApprovalEvent = "Approval"
 )
+
+type Token struct {
```

Comment:
Nit, but let's make sure to add some proper godoc at least to the three types here, making explicit which types should be used when :)

Reply:
6ea4b3c9e